Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -441,6 +454,13 @@ private static void Clean(NugetPackageIdentifier package)
                         selectedDirectories.Add(directory.FullName);
                         break;
                     }
+                    else
+                    {
+                        string versionName = GetNuGetVersionName(directoryName);
+                        // Current API version is NET_Standard_2_0; NET_Standard_4_5 is required to support this NuGet package.
+                        Debug.LogError(string.Format("Current API version is {0}; {1} is required to support this NuGet package.", DotNetVersion, versionName));
```

Comment:
We're actually iterating over all the libraries trying to find a viable one - reporting an error here is too early I believe.

Reply:
that's LogError was removed already.